### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Jonath-z**

### CODE REVIEW

File: src/Exercise6.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "🐏 sheep",
+  "🏇 horse",
+];
+
+const DataHolder = (props) => (
+  <div className="border border-solid border-gray-500 p-5">{props.value}</div>
+);
+
+const Exercise6 = () => {
+  return (
```
Comment (Ntwali-Josue) [inline src/Exercise6.js:17]: I recommend using the wrap all your components in the components folder to improve your folder structure. No need to mix the components in one file as you did here by having `DataHolder` & `Exercise6` in the same file.

**Review by Ntwali-Josue — APPROVED** [review]

File: src/Exercise2.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Exercise2 = () => {
+  return (
+    <div className="flex justify-center items-center mt-16">
+      <button
+        className="bg-blue-500 text-white px-10 py-5 rounded-lg"
+        onClick={() => alert("Hello!! How are you?")}
+      >
+        Click me{" "}
```
Comment (Nkaka23dev) [inline src/Exercise2.js]: Good  job!  however,  why this empty string and bracket?

File: src/Exercise3.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Button = (props) => (
+  <div className="flex justify-center items-center mt-16">
+    <button
+      className="bg-blue-500 text-white px-10 py-5 rounded-lg"
+      onClick={() => alert(props.text)}
+    >
+      Click me{" "}
```
Comment (Nkaka23dev) [inline src/Exercise3.js]: Why this empty strings?

File: src/Exercise3.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Button = (props) => (
+  <div className="flex justify-center items-center mt-16">
+    <button
```
Comment (Nkaka23dev) [inline src/Exercise3.js:5]: Great Job, but I suggested Button  component to be in different file for good practice.

**Review by Nkaka23dev — CHANGES_REQUESTED** [review]

**Review by edson-pro — APPROVED** [review]

**Review by edson-pro — APPROVED** [review]

File: src/Exerice1.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Exercise1 = () => {
+  return (
+    <div className="w-full h-screen flex justify-center items-center bg-yellow-400 text-3xl">
+      hellon world
+    </div>
```
Comment (chrisostome01) [inline src/Exerice1.js]: Try to correct it here, it is supposed to be `hello, world!`,
second, you may take a look at this instruction.
> React. Start by rendering a square with a background color. Inside the square, render "Hello, World!".

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Jonath-z,
I left a comment on the first challenge, please check it out 😉

File: src/Exercise8.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    id: 1,
+    setup: "What's the best thing about a Boolean?",
+    punchline: "Even if you're wrong, you're only off by a bit",
+  },
+  {
+    id: 2,
+    setup: "Why do programmers wear glasses?",
+    punchline: "Because they need to C#",
+  },
+];
```
Comment (chrisostome01) [inline src/Exercise8.js]: The data should be in JSON format

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Jonath-z,
I left a comment on rendering JSON data, you may check it out please.

File: src/Exercise6.js (hunk 1 above)
Comment (Jonath-z) [reply src/Exercise6.js:17]: if I do this now, it will change all the links which other reviewers are using

File: src/Exercise3.js (hunk 4 above)
Comment (Jonath-z) [reply src/Exercise3.js:5]: As this component is only used in this file and for this exercise, I think it is better to only have it here

Comment (Jonath-z) [reply src/Exercise3.js:5]: As this is kind of a mono repo project 🙂, I think as this component concerns only this file and this exercise it is better to have it here only.

**Review by chrisostome01 — APPROVED** [review]
Well done @Jonath-z,
The JSON format looks good.

File: src/Exercise3.js (hunk 4 above)
Comment (Nkaka23dev) [reply src/Exercise3.js:5]: Okay great!

**Review by Nkaka23dev — APPROVED** [review]

**Pull request merged**